index.rst

```diff
@@ -14,35 +14,26 @@ Quick Start
 Here are the basic steps needed to get :ref:`set up <setup>` and contribute a
 patch:
 
-1. Set up and install dependencies.
+1. Install and set up :ref:`Git <vcsetup>` and other dependencies
+   (see the :ref:`Get Setup <setup>` page for detailed information).
 
-   Install :ref:`Git <vcsetup>` and other dependencies.
+2. Fork `the CPython repository on GitHub <cpython github>`_ and
```

Comment:
This should be clear enough, since immediately after it clones from the user account.